### ISSUE TITLE
[eslint-config-fusion#109] suggest adding plugin-jsx-a11y

### ISSUE THREAD

**Issue by chrisdothtml** [fusionjs/fusionjs#289]
This issue was migrated from eslint-config-fusion and was originally reported by @ksheedlo.

---

### Type of issue

Feature request

### Description

Add `eslint-plugin-jsx-a11y` to the config to automatically flag accessibility problems. This could be implemented here in this repo or in eslint-config-uber-universal-stage-3

### Current behavior

Fusion.js does not automatically flag accessibility problems in my apps. Of course, I can still extend this config to add jsx-a11y linting, but I have to remember to do this each time I scaffold a new repo. Moreover, I feel having it in the default configuration would help everyone build more accessible websites using Fusion.js.

### Expected behavior

Fusion.js could use `eslint-plugin-jsx-a11y` to automatically detect accessibility problems and fail the build if one was detected.

### Additional notes

This would be a breaking change, since code exists out in the wild using eslint-config-fusion that would not pass `eslint-plugin-jsx-a11y`.
